django/thunderstore/legal/templates/contracts/contract.html

```diff
@@ -0,0 +1,52 @@
+{% extends 'base.html' %}
+{% load markdownify %}
+{% load cache %}
+
+{% block title %}{{ contract.title }}{% endblock %}
+
+{% block main_content_left %}{% endblock %}
+{% block main_content_right %}{% endblock %}
+{% block content_beginning %}{% endblock %}
+{% block content_end %}{% endblock %}
+
+{% block og_title %}{{ contract.title }} | {{ site_name }}{% endblock %}
+
+
+{% block content %}
+    {% cache 300 templates.contracts.contract version.pk version.datetime_updated contract.datetime_updated %}
+    <div class="card bg-light mt-2">
+        <div class="card-header d-flex justify-content-between gap-2 flex-wrap">
+            <h5 class="mb-0 w-100">{{ contract.title }}</h5>
+            <div class="d-flex justify-content-between gap-2 flex-grow-1 flex-wrap">
+                <div class="d-flex align-items-center gap-2">
+                    <span class="text-muted">Published {{ version.effective_date|date:"Y-m-d" }}</span>
```

Comment:
Just wondering if for legal docs the publishing time also matters? It's plausible for two versions to be published on the same date. Same applies to `contract_history.html`.

Reply:
It's possible, and I originally had the time included, but I didn't want to deal with localizing the timestamp (which is a lesser issue now) and I figured the permalinks + version history still work for disambiguation (as version history is ordered by the date). This is easy to change later if deemed insufficient.